### PR TITLE
navigation bar

### PR DESCRIPTION
**Pull request by mysticlagoona**
Navigation Bar 
- added new icons
 - changed style
Home Screen
 - added a new screen for home
 - added new button for EB
 - added a button for chat

### CODE REVIEW

File: source/components/molecules/MaterialTopTabBarWrapper/MaterialTopTabBarWrapper.js [hunk 1]
```diff
@@ -8,20 +9,24 @@ const MaterialTopTabBarWrapper = props => {
...
+    borderBottomWidth: 2,
+  };
   return (
     <SafeAreaView
       style={{ backgroundColor: '#F8F8F8' }}
       forceInset={{ top: 'always', horizontal: 'never', bottom: 'never' }}
     >
       <MaterialTopTabBar
         {...props}
+        indicatorStyle={indicatorStyle}
```
[the remark this answers is not in the document]
Comment (mysticlagoona) [reply source/components/molecules/MaterialTopTabBarWrapper/MaterialTopTabBarWrapper.js:28]: It's easy to cange colors as borderColor defins the color code :)

[the remark this answers is not in the document]
Comment (TeddyAndersson) [reply source/components/molecules/MaterialTopTabBarWrapper/MaterialTopTabBarWrapper.js:32]: Yeah but all colors are hardcoded in the component now? Feels more natural to just pass props for the borderColor.

I'm thinking of an interface for the component which looks like:
`<MaterialTopTabBarWrapper  tabColors={[color1, color2, color3]} tabBorderColors={[color1, color2, color3]} />`

Then for handling colors we can do: 
`
tabColors[index] || props.defaultColor
tabBorderColors[index] || props.defaultBorderColor
`

[the remark this answers is not in the document]
Comment (TeddyAndersson) [reply source/components/molecules/MaterialTopTabBarWrapper/MaterialTopTabBarWrapper.js:32]: However this is nice to have so if you feel that this is to much of a hustle, then put a task on it so we can revisit it later :)

[the remark this answers is not in the document]
Comment (mysticlagoona) [reply source/components/molecules/MaterialTopTabBarWrapper/MaterialTopTabBarWrapper.js:32]: Props aren't passed like this for MaterialTopBarWrapper. However I will make a todo task to visit later

**Pull request merged**